### PR TITLE
Added environment to avoid unexpected result.

### PR DESCRIPTION
**Pull request by cn007b**
I read all documentation but i hack my dev db when tweak this bundle.)
That's why i decided that it will convenient to specify environment explicitly to emphasize one more time that fixtures will be inserted to test db.

Hope it will be helpful.

### CODE REVIEW

**Comment by fabpot** [comment]
Fixtures are not just for testing. They can also be used to bootstrap dev envs, or even production when deploying the first time.

**Comment by cn007b** [comment]
Yes, agree with you! But, I've found [this](http://prntscr.com/gkafc9) on page: https://github.com/doctrine/DoctrineFixturesBundle/blob/dd52aa9230ba5da5fe02f61fda5833df7bf136b9/Resources/doc/index.rst
How come?

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -109,13 +109,13 @@ line by using the ``doctrine:fixtures:load`` command:
 
 .. code-block:: bash
 
-    php app/console doctrine:fixtures:load
+    php app/console doctrine:fixtures:load -env=test
```
Comment (stof) [inline Resources/doc/index.rst]: this won't work. It must be `-e` or `--env`

**Comment by stof** [comment]
well, the page you linked is seeing the doc for **your** commit.

The fact that it is accessible in the doctrine repository even though it is not reachable by any branch there is due to the fact that all forks are stored as a single git repo in Github, with a special handling of references (emulating the fact that each repo has its own branches).
On a commit page, Github displays in which branches the commit is reachable (after the commit message but before the author, see https://github.com/doctrine/DoctrineFixturesBundle/commit/7bb198c044b798b54e6be37c7929339aa645c3bf). The link you gave does not give any branches allowing to reach your commit.

**Comment by cn007b** [comment]
My fault...
I've deleted my message. Thank you! And sorry for this stupid situation...

**Pull request closed without merging**